packages/app/lib/src/features/home/presentation/home/post.dart

```diff
@@ -198,3 +200,40 @@ class _PostBody extends StatelessWidget {
   }
   // coverage:ignore-end
 }
+
+class _PostInteractables extends HookWidget {
+  const _PostInteractables({
+    required this.post,
+    // Temporary ignore, see <dart-lang/sdk#49025>.
+    // ignore: unused_element
+    super.key,
+  });
+
+  final PostEntity post;
+
+  @override
+  Widget build(BuildContext context) {
+    final thumbsIcon = useState(const Icon(Icons.thumb_up_outlined));
+    return Row(
+      children: [
+        Text(post.numberOfLikes.toString()),
+        IconButton(
+          onPressed: () {
+            if (thumbsIcon.value.icon == Icons.thumb_up_outlined) {
+              thumbsIcon.value = const Icon(Icons.thumb_up_sharp);
+            } else {
+              thumbsIcon.value = const Icon(Icons.thumb_up_outlined);
+            }
+          },
```

Comment:
Is this meant as an animation? 🤣

Reply:
maybe eventually. right now it just updates the icon of the thumb when you click it